packages/directory/app/styles/navi-directory/components/dir-sidebar.less

```diff
@@ -16,6 +16,7 @@
 .dir-sidebar {
   color: @navi-blue-300;
   display: grid;
+  grid-template-rows: fit-content(50px);
```

Comment:
Do we have to update the px when we add elements to the sidebar?

Reply:
No the `50px` refers to the height of one row in the grid.